### PR TITLE
Update jekyll-linkpreview.gemspec

### LINKED ISSUE #21: Not compatible with latest Jekyll version 4.1.0

### PR DESCRIPTION
**Pull request by davidschoi**
fixes #21 

remove jekyll dependency

### CODE REVIEW

File: jekyll-linkpreview.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   end
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "jekyll", ">= 3.5", "< 4.1"
+  spec.add_dependency "jekyll", ">= 3.5"
```
Comment (ysk24ok) [inline jekyll-linkpreview.gemspec]:
```suggestion
  spec.add_dependency "jekyll", ">= 3.5", "< 5.0"
```

**Review by ysk24ok — COMMENTED** [review]
Thank you for the PR.

How about setting an upper bound in order not to use new major version automatically because jekyll 5 may break backward-compatibility and this plugin may not work with the new version.
I've made a code suggestion. Glad if you take a look.

File: jekyll-linkpreview.gemspec (hunk 1 above)
Comment (davidschoi) [reply jekyll-linkpreview.gemspec]: sounds like the right solution. thanks @ysk24ok

**Review by ysk24ok — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**